### DISCUSSION TITLE
Propose new features still possible

### DISCUSSION THREAD

**Discussion by lucacivale** [xamarin/XamarinCommunityToolkit#1763] [Q&A] [answered]
Hello guys,
I want to propose an idea for a new feature. I didn't find a guide or similar regarding features. So I wanted to ask if it's still possible and where I can finde the guideline.

Thanks!

**Comment by pictos** [comment, marked as answer]
Hey, @lucacivale you can write your proposal [here](https://github.com/CommunityToolkit/Maui/discussions/new)
